package.json

```diff
@@ -109,6 +109,8 @@
     "tlhunter-sorted-set": "^0.1.0"
   },
   "devDependencies": {
+    "@eslint/eslintrc": "^3.1.0",
```

Comment:
It looks like the old dependency is also still installed if I understand this properly.

Reply:
@rochdev `@eslint/eslintrc` is not `@eslint/eslint`, if that's what you're talking about? `@eslint/eslintrc` is a compatibility layer allowing new-style config to be generated from old-style config.

See https://github.com/DataDog/dd-trace-js/pull/4759/files/d049b1eb11fc33c065fbe6a370318fbe74fe8080#diff-9601a8f6c734c2001be34a2361f76946d19a39a709b5e8c624a2a5a0aade05f2R7 and usages of it in that file.

Reply:
I misunderstood that upgrading eslint was part of this PR when it's not.